packages/notifi-dapp-example/src/components/WalletSelectModal.tsx

```diff
@@ -14,16 +14,7 @@ export const WalletSelectModal: FC<WalletSelectModalProps> = ({
 }) => {
   const { wallets } = useWallets();
   const availableWallets = objectKeys(wallets);
-  let walletsRequiredForGMX: typeof availableWallets = [
-    'metamask',
-    'coinbase',
-    'rabby',
-    'walletconnect',
-    'binance',
-    'okx',
-    'rainbow',
-    'zerion',
-  ];
+  let walletsRequiredForGMX: typeof availableWallets = ['xion'];
```

Comment:
Ah ... well, I understand this one is specifically for Xion dapp. 
However, this is good. But do you think it might be better to separate this PR into two:
1. for the generic sections which can be be use commonly in the future. (ex. notifi-wallet-provider) --> against main branch 
2. for others, we can make it specifically for Xion